frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_card_on_cluster_overview_ODC-4922.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Add quick starts card on cluster overview
+	As a user, I want to easily be able to access and discover Quick Starts from the Cluster Overview in administrator view.
```

Comment:
```suggestion
	As a user, I should get easy access to discover Quick Starts from the Cluster Overview option in Administrator perspective
```

Reply:
No, I think it would be:
```suggestion
	As a user, I should be able to access and discover Quick Starts from the Cluster Overview in the Administrator view.
```

Reply:
Updated